src/features/Staff/pages/StaffPage/StaffPage.tsx

```diff
@@ -45,6 +45,16 @@ export const StaffPage: VFC = () => {
     return [];
   }, [staff]);
 
+  const links = staff != null ? staff.links.map(link => (
+    <Link
+      key={link.href}
+      href={link.href}
+      textTransform="capitalize"
+    >
+      {link.label}
+    </Link>
+  )) : [];
```

Comment:
I think this example offers better readability, because you don't need to scan down whole code block to know second case of condition. What do you think? 🤔
```suggestion
  const links = (staff != null ? staff.links : []).map(link => (
    <Link
      key={link.href}
      href={link.href}
      textTransform="capitalize"
    >
      {link.label}
    </Link>
  ));
```